.github/workflows/pythonbuild.yml

```diff
@@ -141,7 +147,7 @@ jobs:
         run: |
           pip install uv
           make setup-global-uv
-          uv pip install --system --force-reinstall "${{ matrix.pandas }}"
+          uv pip install --system --force-reinstall "${{ matrix.pandas }}" "${{ matrix.numpy }}"
```

Comment:
`numpy` should be a member of `matrix` (similar to [pandas](https://github.com/flyteorg/flytekit/blob/master/.github/workflows/pythonbuild.yml#L124)).

Reply:
we don't want the cartesian product of all four combinations tho right?
```
- pandas<2.0.0, numpy<2.0.0
- pandas<2.0.0, numpy>=2.0.0
- pandas>=2.0.0, numpy<2.0.0
- pandas>=2.0.0, numpy>=2.0.0
```